docs/guide/README.md

```diff
@@ -1,94 +1,94 @@
 
-Most of the Vue Storefront configuration (like active theme, backend API addresses, multistore setup, etc.) is done through its config file that can be found under the `config` folder. The `default.json` file contains contains the default setup.
+## Vue Storefront config file
 
-For your own implementation you should create a `local.json` file in the same directory and include fields from `default.json` that you want to override. These two files will be merged in favor of `local.json` during a build process. If you use installer to set up your Vue Storefront instance, it'll generate proper config files.
+Most of the Vue Storefront configuration (like the active theme, backend API addresses, multistore setup etc) is done through its config file that can be found under `config` folder. The `default.json` file contains all the default setup.
+
+For your own implementation, you should create a `local.json` file in the same directory and include fields from `default.json` that you want to override. This two files will be merged in favour of `local.json` during a build process. If you will use the installer to set up your Vue Storefront instance it'll generate proper config files.
 
 ## Building themes in Vue Storefront
 
-While making themes in Vue Storefront, in most cases, all you need to do is create your own HTML and CSS markup. All required business logic is exposed by Core with its core modules and can be easily injected into any of the theme components. The mechanism of injecting core business logic into themes is very simple. We are using Vue.js mixins to keep upgradable business logic in the core.
+While making themes in Vue Storefront in most cases all you need to take care of is creating your own HTML and CSS markup. All required business logic is exposed by the core with its core modules and can be easily injected into any of the theme components.
+The mechanism of injecting core business logic into themes is very simple. We are using Vue.js mixins to keep upgradable business logic in the core.
 
-Business logic from the core component can be easily injected into any theme component as a Vue.js mixin. We can inject it into any of our theme components just by importing and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
+Business logic from core component can be easily injected into any theme component as a Vue.js mixin. We can easily inject it into any of our theme components just by importing it and adding it as a mixin ( eg `mixins: [Microcart]` is what you need to use core Microcart logic). This is all you need to make use of core business logic inside your theme. With this approach, we can easily ship updates to all core components without breaking your shop.
```

Comment:
why was "the" removed from "the core component"? It was correct English. Please change back.